convert_collection_to_ndjson.py

```diff
@@ -0,0 +1,12 @@
+import argparse
+from pathlib import Path
+
+parser = argparse.ArgumentParser()
+parser.add_argument('collection', type=Path)
+args = parser.parse_args()
+
+with args.collection.open() as f:
+    data = f.read().replace('\n', '') + '\n'
+
+with args.collection.with_suffix('.ndjson').open('w') as f:
+    f.write(data)
```

Comment:
I prefer `json` to read/write the data; shouldn't have serialization issues since the collection file is already valid json where all the dates are strings.
```suggestion
with args.collection.open() as f:
    data = json.load(f)

with args.collection.with_suffix('.ndjson').open('w') as f:
    json.dump(data, f)
```

I imagine we'll eventually want to loop over all of our collection jsons and write them to a single ndjson file, but I'm content to leave that until we add a second collection.

Reply:
Sounds good, that was the other implementation that I considered.